packages/util/src/signature.ts

```diff
@@ -8,6 +8,7 @@ export interface ECDSASignature {
   v: bigint
   r: Buffer
   s: Buffer
+  recovery: bigint
 }
```

Comment:
Another short review round 🙂: can you please add some code docs to this interface?